client/js/components/day/gig.js

```diff
@@ -0,0 +1,50 @@
+var React = require('react');
+var Staff = require('./staff-card');
+
+var Gig = React.createClass({
+
+  // information: object full of gig info (like location)
+  // staff: object with information about the staffing needs
+    // and the approved employees.
+  getDefaultProps: function(){
+    return {
+      information: {},
+      staff: {}
+    };
+  },
+
+  render: function(){
+    return (
+      <div>
+        <GigInformation info={this.props.information} />
+        <Staff staff={this.props.staff} />
+      </div>
+    );
+  }
+
+});
+
+var GigInformation = React.createClass({
+
+  getDefaultProps: function(){
+    return {
+      info: {}
+    };
+  },
+
+  render: function(){
+    var info = this.props.info;
+
+    return (
```

Comment:
This is solid work considering specific event requirements that we will want to include in notifications. However, I believe this and other high-level event information belongs on the week view where events and their details are the primary focus. The primary focus of the day view is to determine where employees should be placed between different events, or if they should be placed at all.

We can relocate the information once our backend schema is solidified.